lesson_16/objects/objects_app/src/main/java/com/codedifferently/footballteamobject/FootballTeam.java

```diff
@@ -0,0 +1,106 @@
+package com.codedifferently.footballteamobject;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FootballTeam {
+
+  public static class DuplicatePlayerException extends Exception {
+    public DuplicatePlayerException(String message) {
+      super(message);
+    }
+  }
+
+  private int playerCount;
+  private int jerseyNumber50AndOver;
```

Comment:
Personally, I’d combine `jerseyNumber50AndOver` and `jerseyNumberUnder50` into a single variable, it feels a bit redundant. That said, the way you’ve implemented it works perfectly fine.